public/app/features/plugins/admin/hooks/usePluginDetailsTabs.tsx

```diff
@@ -63,6 +64,16 @@ export const usePluginDetailsTabs = (
       });
     }
 
+    if (isPublished && plugin?.details?.screenshots && plugin?.details?.screenshots?.length > 0) {
```

Comment:
nit: optional chain already takes care of validating the attributes so the first validation is redundant.
```suggestion
    if (isPublished && plugin?.details?.screenshots?.length > 0) {
```

Reply:
this causes a type error, because if plugin, details or screenshots is undefined, than it tries to compare undefined > 0.

Reply:
just keep `plugin?.details?.screenshots && plugin?.details?.screenshots?.length` `0` is a falsy value